client/app/components/packages/rwcds-form/proposed-project-actions.hbs

```diff
@@ -0,0 +1,85 @@
+{{#let @form as |Form|}}
+  <Form.Section @title="Proposed Actions">
+    <p>
+      Listed below are the actions involved in this project:
+    </p>
+
+  <div data-test-section="proposed-project-actions">
+    {{#each @form.saveableChanges.affectedZoningResolutions as |affectedZoningResolution|}}
+      <Form.SaveableForm
+        @model={{affectedZoningResolution}}
+        @validators={{array @validations.SaveableAffectedZoningResolutionFormValidations @validations.SubmittableAffectedZoningResolutionFormValidations}}
+        as |affectedZoningResolutionForm|
+      >
+        <Packages::RwcdsForm::ProposedProjectActionsEditor
+          @landUseAction={{affectedZoningResolutionForm.saveableChanges}}
+          @form={{affectedZoningResolutionForm}}
```

Comment:
```
@form=affectedZoningResolutionForm
```
affectedZoningResolutionForm has `affectedZoningResolutionForm.saveableChanges` (i want to make a property `data`)